### PR TITLE
Add prometheus listen address parameter to config

### LINKED ISSUE #311: Add a parameter to config specifying the port for Prometheus exporter

### PR DESCRIPTION
**Pull request by Justarone**
Closes #311  (notice, that I added address instead of port only, because it's more extendible and was easier to add)

### CODE REVIEW

File: bob-common/src/configs/node.rs [hunk 1]
```diff
@@ -150,6 +152,14 @@ impl MetricsConfig {
         }
     }
 
+    pub(crate) fn prometheus_addr(&self) -> &str {
+        &self.prometheus_addr
+    }
+
+    pub(crate) fn default_prometheus_addr() -> String {
+        "0.0.0.0:9000".to_owned()
```
[the remark this answers is not in the document]
Comment (Justarone) [reply bob-common/src/configs/node.rs:160]: @idruzhitskiy  in this doc 9090 is default port for prometheus server, not exporter. 9000 - is default value of prometheus exporter in metrics-exporter-prometheus lib, so I decided to set it explicitly to make rfurther code eading easier

**Comment by ikopylov** [comment]
LGTM

Please, merge `master` into this branch and resolve conflicts

**Comment by Justarone** [comment]
Didn't change anything since last review, only merged master.

**Comment by ikopylov** [comment]
@pyakushin can you take a look?

**Pull request merged**